Dockerfile

```diff
@@ -32,14 +30,14 @@ RUN echo "@edge https://nl.alpinelinux.org/alpine/edge/main" >> /etc/apk/reposit
     imagemagick@edge \
     ca-certificates \
  && npm install -g npm@3 && npm install -g yarn \
- && bundle install --deployment --without test development \
- && yarn --ignore-optional --pure-lockfile \
- && yarn cache clean \
```

Comment:
Why remove `yarn cache clean` task? for https://github.com/rails/webpacker/issues/405 ?